src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```
@@ -164,6 +164,10 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NativeSystemLibrary Include="crypto" />
     </ItemGroup>
 
+    <ItemGroup Condition="'$(UseSystemBrotli)' != 'false' and Exists('$(IlcSdkPath)nonportable.txt')">
+      <NativeSystemLibrary Include="brotlienc;brotlidec;brotlicommon" />
```

```suggestion
      <NativeSystemLibrary Include="brotlienc" />
      <NativeSystemLibrary Include="brotlidec" />
```
brotlicommon is only used for static linking: https://github.com/dotnet/runtime/pull/107166#discussion_r1739122983

Also, we may want to follow the style of other `NativeSystemLibrary` items.